lib/ahoy/intercom/tracker.rb

```diff
@@ -1,12 +1,13 @@
 require 'intercom'
 require 'ahoy/intercom/client'
+require 'logger'
 
 module Ahoy
   module Intercom
     module Tracker
       include Client
 
-      def track(event_name, properties = {})
+      def save_to_intercom(event_name, properties = {})
```

Comment:
no, it still should be "track_event", please check Ahoy documentation about creating new custom stores.